lib/taploFmt.nix

```diff
@@ -0,0 +1,24 @@
+{ taplo
+, mkCargoDerivation
+}:
+
+{ taploExtraArgs ? ""
```

Comment:
Maybe we can make the default here `"--config ./toplo.toml"` and spare callers from needing to pass it in themselves (but they are welcome to override it completely)

Reply:
Just now getting back to this. 

The only downside of using `"--config ./taplo.toml"` as the default is that if the user doesn't have that file, it will show an error and then use the default options. Maybe something cool we could do is add a field called `useConfig` which will search the file tree for a `.taplo.toml`/`taplo.toml`(more info on that here https://taplo.tamasfe.dev/configuration/file.html) and if it doesn't find one then it will show a more helpful error instead of passing over the error and continuing, potentially causing the user to misinterpret the check as successfully using their config file.

Reply:
I use the `taploExtraArgs` in the examples to make it more clear to new users where they can find the `taplo.toml` and make adjustments. If you'd like maybe I could also include a link to the configuration documentation above for other options?

Reply:
Gone ahead and added the link to the docs in the API section for the function

Reply:
I'm good leaving the default args blank here in that case! I also tweaked the examples to comment out the taplo configs with a comment saying it's possible (but not required) to define it!